docs/comparison_with_webpacker.md

```diff
@@ -4,20 +4,103 @@ If you're already using [`webpacker`](https://github.com/rails/webpacker), you m
-- `jsbundling-rails` uses the standard configuration format for your bundler of choice.
-  `webpacker`/`shakapacker` has an optional configuration layer on top of `webpack`'s configuration. You don't have to use it. The only requirement of webpacker is that your webpack configuration must produce a manifest.
-- `jsbundling-rails` works with the standard [`actionview` asset helpers](https://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html).
-  `webpacker`/`shakapacker` provides view helpers with an almost identical API.
-- `webpacker`/`shakapacker` can be used as a complete replacement for [`sprockets`](https://github.com/rails/sprockets), and in that setup you can stop loading the `sprockets/railtie` in your application. What you produce in the webpack ecosystem is what is sent to the browser.
-  `jsbundling-rails` (as well as [`css-bundling-rails`](https://github.com/rails/cssbundling-rails)) works in conjunction with `sprockets`. Because of this, you need to be sure not to double fingerprint your assets in both `webpack` and `sprockets`. You might also have issues with source maps due to double fingerprinting of output files.
-- `webpacker`/`shakapacker` supports using the `webpack-dev-server` for hot reloading. HMR enables you to see your changes in the browser almost immediately as you make them, usually without the need to refresh the page or lose your application state. 
-  `jsbundling-rails` hands over static files to `sprockets`, so hot reloading is not supported. To load any JavaScript changes, you'll need to do a local-state-clearing full page refresh.
-- `webpacker`/`shakapacker` delegates asset processing entirely to the external nodejs tooling.
-  `jsbundling-rails` still relies on `sprockets` to output the final `public` assets and create the associated manifest.
+<table>
+  <tr><th>
+
+  `jsbundling-rails`
+  </th><th>
+
+  `webpacker`/`shakapacker`
+  </th></tr><tr><td>
+  <!-- jsbundling: -->
+
+  A simple integration composed only of installed files and a couple of new rake tasks.
+
+  </td><td>
+  <!-- webpacker/shakapacker: -->
+
+  Provides view helpers and a customizable webpack config that include support for the most important advanced features of webpack: code splitting and HMR.
+
+  You don't need to use the `webpacker` webpack config. You could choose only to use the view helper portion of `webpacker`/`shakapacker`.
+
+  </td></tr><tr><td>
+  <!-- jsbundling: -->
+
+  Can be used with multiple bundlers.
+
+  Currently `esbuild`, `rollup`, and `webpack` are supported out of the box, but anything that can put a bundle into `app/assets/builds` could be configured to work with it.
+
+  </td><td>
+  <!-- webpacker/shakapacker: -->
+
+  Built specifically to integrate with `webpack`. That allows `webpacker`/`shakapacker` to provide support for HMR and code splitting.
```

Comment:
https://github.com/shakacode/shakapacker#esbuild-loader-configuration

SWC and Esbuild are currently supported.

Reply:
@justin808 How do you suggest rephrasing?

```markdown
`webpacker` was originally built specifically to integrate with `webpack`.  That allows
`webpacker` to provide support for HMR and code splitting.

`shakapacker` also supports `SWC` and `esbuild` _(experimental as of October 2022)_ with
integrations into some of their advanced features.
```

That feels maybe a little awkward to me, but since that `tr` references integrations like HMR (which isn't supported by `esbuild`, right?), I didn't want to be misleading on that point.

Perhaps I can rebase on top of a branch/PR where you make that textual change (in the original list format)?  I was trying to keep this commit purely about formatting, as much as possible, with no content changes. 🙂

Reply:
I think the change above sounds fine.